feedbacks.md

```diff
@@ -0,0 +1,33 @@
+# Feedbacks Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/core/feedbacks**   
```

Comment:
Minor recommendation, I'd recommend putting this endpoint under `/api/tools/feedbacks` alongside the `/api/tools/itemrequests`...as both of these endpoints are similar in that they are a way for authenticated or non-authenticated users to communicate with site admins, etc.